benchmarks/actors/src/main/scala/org/renaissance/actors/AkkaUct.scala

```diff
@@ -2,48 +2,55 @@ package org.renaissance.actors
 
 import edu.rice.habanero.actors.AkkaActorState
 import edu.rice.habanero.benchmarks.uct.UctAkkaActorBenchmark
+import org.renaissance.Benchmark
+import org.renaissance.Benchmark._
+import org.renaissance.BenchmarkContext
 import org.renaissance.BenchmarkResult
-import org.renaissance.Config
-import org.renaissance.EmptyResult
+import org.renaissance.BenchmarkResult.Validators
 import org.renaissance.License
-import org.renaissance.RenaissanceBenchmark
-import org.renaissance.Benchmark._
 
 @Name("akka-uct")
 @Group("actors")
 @Summary("Runs the Unbalanced Cobwebbed Tree actor workload in Akka.")
 @Licenses(Array(License.MIT))
 @Repetitions(24)
-class AkkaUct extends RenaissanceBenchmark {
+@Parameter(name = "loop_count", defaultValue = "10")
+// Work around @Repeatable annotations not working in this Scala version.
+@Configurations(
+  Array(
+    new Configuration(name = "test", settings = Array("loop_count = 2")),
+    new Configuration(name = "jmh", settings = Array("loop_count = 1"))
+  )
+)
+final class AkkaUct extends Benchmark {
 
   // TODO: Consolidate benchmark parameters across the suite.
   //  See: https://github.com/renaissance-benchmarks/renaissance/issues/27
```

Comment:
I'm not sure. It was you who opened it :-)

This PR provides support for named configurations and (at the moment) its supports one configuration tag (`$cpu.count`) which expands to number of available CPUs.

But does it count as having consolidated the benchmark parameters? If yes, then I can remove the comments, otherwise it should be removed on a case-by-case basis.

Reply:
I would say that consolidating the parameters also means defining reasonable configurations for each benchmark and possibly also changing the default values. I would keep track of this, either in #27 or in another issue.